lib/kafka/consumer.rb

```diff
@@ -199,7 +201,7 @@ def each_message(min_bytes: 1, max_wait_time: 5)
               end
             end
 
-            mark_message_as_processed(message)
+            mark_message_as_processed(message) if automatically_mark_as_processed
```

Comment:
I think it would make sense to only add this option to `#each_batch` – I'd like to see an actual use case where it would make sense to use it with `#each_message` before adding more API surface.